sycl/test-e2e/LLVMIntrinsicLowering/sub_byte_bitreverse.cpp

```diff
@@ -13,8 +13,9 @@
 // Ensure that SPV_KHR_bit_instructions is disabled so that translator
 // will lower llvm.bitreverse.* intrinsics instead of relying on SPIRV
 // BitReverse instruction.
-// Also build executable with SPV dump.
-// RUN: %{build} -o %t.out -O2 -Xspirv-translator --spirv-ext=-SPV_KHR_bit_instructions -fsycl-dump-device-code=%t.spvdir
+// Also build executable with SPV dump.  Use -fno-sycl-allow-device-dependencies to
```

Comment:
what does this `-fno-sycl-allow-device-dependencies` flag do?  I see it in a few tests, but it's not documented.

Reply:
-fsycl-allow-device-dependencies allows device images to have dependencies to other device images.   With the -fno-sycl-allow-device-dependencies option (the default behavior), device images are constructed by sycl-post-link to be self-contained.  A side-effect of the new behavior in this test case with -fsycl-allow-device-dependencies is that multiple SPV files are generated.  The analysis of the SPV file in this testcase only supports one SPV file, thus -fno-sycl-allow-device dependencies was added to prepare for the possibility of the default being switched.